content/articles/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/index.md

```diff
@@ -0,0 +1,169 @@
+
+![plots](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-one.jpeg)
+
+Symbolic math in live editor lets you interactively update and display symbolic math computations alongside Matlab code formatted text, equations, and images which can be published as executable live scripts, pdfs, or html documents. While working with analytical, you can get suggestions for the next step. These suggestions help you insert and execute function calls or live editor tasks directly into live scripts.
+In addition to providing exact rational math, the symbolic toolbox provides variable precision allowing algorithms to run in a higher or lower position than Matlab's in-built double. It also includes units for working with physical quantities and performing dimensional analysis.
+
+![Units for working with physical quantities](/engineering-education/how-to-use-symbolic-math's-toolbox-in-matlab-to-analyze-the-fourier-series/symbolic-two.jpeg)
+
+This toolbox is widely applied in many engineering and scientific application such as optimization, where symbolic expressions of exact gradient and Hessians improves accuracy and speed of optimization.
+In non-linear control design, symbolic linearization and state-space description improve the speed of re-calculation at any operating point during execution. Furthermore, you can integrate symbolic results with Matlab and Simulink applications by converting symbolic expressions into numeric Matlab functions, Simulink, and Simscape blocks.
+Now, all these applications discussed above were to give you an eye-sight on the wide application of this toolbox. All these are not discussed here. Here we just major on using the toolbox to solve Fourier series problems.
+
+### How to use the symbolic toolbox
+This toolbox is enabled in Matlab using the function `syms`. If you have an expression `x=2*a+b` and you try to execute it in Matlab, you get an error message `undefined function or variable a` as shown below:
+```Matlab
+x = 2*a + b
+```
+output
+
+```matlab
+Undefined function or variable 'a'.
+```
+When using the `syms` function, the variable `x` is saved without the error message. The idea here is, when using the symbolic toolbox, you first define the symbolic variables. Symbolic variables are the undefined variables in an equation. For our expression above, our symbols are `a` and `b`, so we first define these variables using the symbolic function `syms`.
+```Matlab
+syms a b
+```
+After defining the symbols and re-run the code above, our variable is stored. So we will have;
+```Matlab
+syms a b
+x = 2*a + b
+```
+Output:
+```Matlab
+x =
+ 
+2*a + b
+```
+
+### Solving Fourier series using the symbolic toolbox
+Let's say we have a Fourier transform shown below:
+
+$$\int e^{(\frac{-t}{2})}u(t) e^{-jwt}dt$$
+
```

Comment:
I have added the ajax script and so the symbols are properly displayed and no need to use the tool you recommended

Reply:
> I have added the ajax script and so the symbols are properly displayed and no need to use the tool you recommended

Hello @linetachieng . I am really sorry for that. Has your article ever been published without you using the script. If so, you can remove the script because it's displaying properly without the script.